Apps/AccountDataAccess/src/Patient/PatientMappings.cs

```diff
@@ -0,0 +1,155 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.AccountDataAccess.Patient
+{
+#pragma warning disable SA1600 // Disables documentation for internal class.
+    using System.Collections.Generic;
+    using AutoMapper;
+    using HealthGateway.AccountDataAccess.Patient.Api;
+
+    internal class PatientMappings : Profile
+    {
+        private const string Delimiter = " ";
+
+        public PatientMappings()
+        {
+            this.CreateMap<PatientIdentity, PatientModel>()
+                .IncludeAllDerived()
+                .ForMember(d => d.Birthdate, opts => opts.MapFrom(s => s.Dob))
+                .ForMember(d => d.CommonName, opts => opts.MapFrom(s => ExtractPreferredName(s)))
+                .ForMember(d => d.LegalName, opts => opts.MapFrom(s => ExtractLegalName(s)))
+                .ForMember(d => d.PhysicalAddress, opts => opts.MapFrom(s => ExtractPhysicalAddress(s)))
+                .ForMember(d => d.PostalAddress, opts => opts.MapFrom(s => ExtractPostalAddress(s)))
+                .ForMember(d => d.IsDeceased, opts => opts.MapFrom(s => s.HasDeathIndicator));
+        }
+
+        private static Name ExtractPreferredName(PatientIdentity patientIdentity)
+        {
+            string? preferredName = AddToString(string.Empty, patientIdentity.PreferredFirstName);
```

Comment:
using the `AddToString` method here doesn't add any value, the first test is if `string.IsNullOrWhitespace`, which covers `string.Empty` thus it would automatically just accept the value (`PrefferedFirstName`).

So you may as well just use `patientIdentity.PreferredFirstName` directly.

```suggestion
            string? preferredName = patientIdentity.PreferredFirstName;
```

Reply:
The first parameter in AddToString is called 'existingString'.  When adding 'PreferredFirstName', there is no value for 'existingString'.  So when passing 'existingString' is null or empty, then just return 'PreferredFirstName'.  If 'PreferredFirstName' is null or empty, then return 'existingString'.  If 'existingString' and 'PreferredFirstName' are not null or empty, then combine the two and return.  This has to be called for PreferredSecondName, PreferredThirdName, etc.

<img width="945" alt="Screenshot 2023-05-01 at 10 35 12 AM" src="https://user-images.githubusercontent.com/58790456/235498115-1e1e45da-ef97-4afe-bfab-155b89922f2b.png">